styles.css

```diff
@@ -1,4 +1,6 @@
 
 @import url(./components/root/root.css);
 @import url(./components/avatars/avatars.css);
```

Comment:
Keep folder naming consistent, as you have used `Alerts` and 'avatars`. Keep either capital or small.

Reply:
Done thanks for the Review buddy